### DISCUSSION TITLE
DOCS - Incorrect documentation on pre-upgrade checker configuration

### DISCUSSION THREAD

**Discussion by xorinzor** [longhorn/longhorn#8327] [General]
I didn't know what Issue category it would fall under, so I'll just put it here.

The documentation for installing with ArgoCD seems outdated. The `values` makes use of `helmPreUpgradeCheckerJob.enabled: false` whereas this has changed to `preUpgradeChecker.jobEnabled: false` in `1.6.0+`.

Old documentation can be [read in step 3, here](https://longhorn.io/docs/1.6.1/deploy/install/install-with-argocd/#installing-longhorn).

This has been noted in issue #6415 and [commented on by 2 others](https://github.com/longhorn/longhorn/issues/6415#issuecomment-1936996757), but doesn't appear to have triggered any action.

**Comment by derekbit** [comment]
cc @yangchiu @mantissahz

**Comment by innobead** [comment]
@derekbit Let's convert this to an issue and assign to @yangchiu . Thanks.

**Comment by mantissahz** [comment]
@derekbit I have converted this to an issue.
